server/channels/app/export.go

```diff
@@ -163,6 +171,19 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 	return nil
 }
 
+func (a *App) exportAttachments(ctx request.CTX, attachments []imports.AttachmentImportData, outPath string, zipWr *zip.Writer, totalExportedFiles *int, totalFiles int) *model.AppError {
```

Comment:
The int pointer works, but it's not easy to read. Shouldn't `totalExportedFiles` always be `len(attachments)`?

Reply:
`totalExportedFiles` is the number of successfully export files, not `len(attachments)`

Reply:
I'm going to +1 that the int pointer makes things just a bit harder to read, but it's only a nit for me.

Reply:
Does the destination matter? `totalExportedFiles`totalExportedFiles can be a local variable inside of `exportAttachments`. If `exportAttachments` didn't return an error, we `totalExportedFiles` is always `len(attachments)`, right?

Reply:
`totalExportedFiles` was not always `len(attachments)` because it was the accumulated number of attachments exported across the 3 attachment types. 

In the latest commit I just decided to separate out the progress across the attachment types and removed the pointer and `totalFiles`